contexts/InvoiceContext.tsx

```
@@ -62,301 +63,338 @@ type InvoiceContextProviderProps = {
+  const { getValues, reset } = useFormContext<InvoiceType>();
+
+  // Variables
+  const [invoicePdf, setInvoicePdf] = useState<Blob>(new Blob());
+  const [invoicePdfLoading, setInvoicePdfLoading] = useState<boolean>(false);
+
+  // Saved invoices
+  const [savedInvoices, setSavedInvoices] = useState<InvoiceType[]>([]);
+
+  useEffect(() => {
+    let savedInvoicesDefault;
+    if (typeof window !== undefined) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)